repository/src/main/java/com/pentaho/pdi/ws/RepositorySyncWebService.java

```diff
@@ -0,0 +1,153 @@
+/*!
+ * Copyright 2010 - 2017 Pentaho Corporation.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.pentaho.pdi.ws;
+
+import com.pentaho.pdi.messages.Messages;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.pentaho.di.core.exception.KettleException;
+import org.pentaho.di.core.plugins.PluginRegistry;
+import org.pentaho.di.core.plugins.RepositoryPluginType;
+import org.pentaho.di.core.xml.XMLParserFactoryProducer;
+import org.pentaho.di.repository.RepositoriesMeta;
+import org.pentaho.di.repository.RepositoryMeta;
+import org.pentaho.platform.engine.core.system.PentahoSystem;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+
+import javax.jws.WebService;
+import java.io.Serializable;
+import java.io.StringBufferInputStream;
+
+@WebService( endpointInterface = "com.pentaho.pdi.ws.IRepositorySyncWebService", serviceName = "repositorySync",
+    portName = "repositorySyncPort", targetNamespace = "http://www.pentaho.org/ws/1.0" )
+public class RepositorySyncWebService implements IRepositorySyncWebService, Serializable {
+
+  private static final long serialVersionUID = 743647084187858081L; /* EESOURCE: UPDATE SERIALVERUID */
+
+  private static Log log = LogFactory.getLog( RepositorySyncWebService.class );
+  private static final String SINGLE_DI_SERVER_INSTANCE = "singleDiServerInstance";
+
+  public RepositorySyncStatus sync( String repositoryId, String repositoryUrl ) throws RepositorySyncException {
+    boolean singleDiServerInstance =
+        "true".equals( PentahoSystem.getSystemSetting( SINGLE_DI_SERVER_INSTANCE, "true" ) ); //$NON-NLS-1$ //$NON-NLS-2$
+    if ( singleDiServerInstance ) {
```

Comment:
Since the 'singleDiServerInstance' var is only used once, these four lines can be compressed to one.
Like this: `if (Boolean.parseBoolean(PentahoSystem.getSystemSetting( SINGLE_DI_SERVER_INSTANCE, "true" )) {
return RepositorySyncStatus.SINGLE_DI_SERVER_INSTANCE;
}`

Reply:
2 lines that is